Documentation/applications/testing/index.rst

```diff
@@ -26,6 +26,8 @@ with little value as usage examples.
 - drivertest - vela cmocka driver test
 - fatutf8 - FAT UTF8 test
 - fdsantest - vela cmocka fdsan test
+- fmemopen - fmemopen() test
```

Comment:
@michallenc have we implemented fmemopen?

Reply:
pull requests created https://github.com/apache/nuttx/pull/11011 https://github.com/apache/nuttx-apps/pull/2155